model.go

```diff
@@ -6,18 +6,31 @@ import (
 	"os"
 )
 
-type Param struct {
+type Model []Layer
```

Comment:
__[gometalinter]__
* [golint] exported type Model should have comment or be unexported